osc/commandline.py

```diff
@@ -7868,9 +7868,11 @@ def do_info(self, subcmd, opts, *args):
         """
 
         args = parseargs(args)
-        pacs = Package.from_paths(args)
-
-        for p in pacs:
+        for pdir in args:
+            if (Path(pdir) / ".osc" / "_package").is_file():
+                p = Package(pdir)
+            else:
+                p = Project(pdir, getPackageList=False, wc_check=False)
```

Comment:
There's a `Store` object that encapsulates working with `.osc` and has several useful methods we could use here. Also, `get_store()` determines the current package/project type and returns either `obs_scm.Store` or `git_scm.GitStore`.

```
from .store import get_store
store = get_store(pdir)

if store.is_package:
    ...
else:
    ...

Reply:
Done (slightly differently)